github/repos_test.go

```diff
@@ -926,3 +926,43 @@ func TestPullRequestReviewsEnforcementRequest_MarshalJSON_nilDismissalRestirctio
 		t.Errorf("PullRequestReviewsEnforcementRequest.MarshalJSON returned %+v, want %+v", string(json), want)
 	}
 }
+
+func TestRepositoriesService_ListAllTopics(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/repos/o/r/topics", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		fmt.Fprint(w, `{"names":["go", "go-github", "github"]}`)
+	})
+
+	want := &Topics{Names: &[]string{"go", "go-github", "github"}}
+	topics, _, err := client.Repositories.ListAllTopics(context.Background(), "o", "r")
+	if err != nil {
+		t.Errorf("Repositories.ReplaceAllTopics returned error: %v", err)
+	}
+
+	if !reflect.DeepEqual(topics, want) {
```

Comment:
Move `want` just above this line. It's pretty common to get the value of `got`, then define `want` and check if they're not equal. See tests above.

```Go
want := ...
if !reflect.DeepEqual(topics, want) {
```